cli/pcluster/config/param_types.py

```diff
@@ -434,6 +434,26 @@ def get_cfn_value(self):
 # that require some custom action during CFN/file conversion
 
 
+class ExtraJsonParam(JsonParam):
+    """Class to manage extra_json configuration parameters."""
+
+    def to_cfn(self):
+        """Convert param to CFN representation, if "cfn_param_mapping" attribute is present in the Param definition."""
+        if "cluster" in self.value:
```

Comment:
We need to check the `self.value` because `NoneType` is not iterable (sorry I missed it in my previous comment).
```python
if self.value and "cluster" in self.value:
```

Reply:
changed